multiaccounts/accounts/database_test.go

```diff
@@ -302,6 +302,17 @@ func TestKeypairs(t *testing.T) {
 	}
 	require.Equal(t, true, locked)
 
+	// Test detleting accounts (addresses) for a certain keycard
```

Comment:
Why not another test function?

Reply:
@stefandunca it could be separate test for each function from `TestKeypairs` test. But added it here cause `TestKeypairs` tests all functionalities we have for keypairs, that's why I added it here.

Reply:
Thanks for the explanations. It is usually easier to debug and maintain when you have separate unit-tests.